src/components/VLogoLoader/VLogoLoader.vue

```diff
@@ -5,11 +5,12 @@
     :class="{
       [$style.loading]: status === 'loading' && !prefersReducedMotion,
       'h-10 w-10': autoResize,
+      'h-12 w-12': !autoResize,
     }"
     aria-hidden="true"
     :data-prefers-reduced-motion="prefersReducedMotion"
     data-testid="logo-loader"
-    class="inline-flex h-12 w-12 items-center justify-center rounded p-3 md:h-12 md:w-12"
+    class="inline-flex items-center justify-center rounded p-3 md:h-12 md:w-12"
```

Comment:
Do the `md:` height and width classes not need to be moved?

Reply:
Moved to the beginning of the class list? I think the prefixed classes are always at the end.